### PR TITLE
Feature/debug while running

### PR DESCRIPTION
**Pull request by jfperusse**

### CODE REVIEW

File: src/aircode.ts [hunk 1]
```diff
@@ -304,21 +355,28 @@ export class AirCode implements vscode.FileSystemProvider {
...
             parent.promises.clear();
             parent.projectName = undefined;
-            vscode.window.showErrorMessage(`Connection lost to ${host}.`);
+
+            statusBarResource?.dispose();
+            statusBarResource = vscode.window.setStatusBarMessage(`Connection lost to ${host}`);
             if (event.code != CloseEventCode.IncompatibleVersion) {
-                vscode.window.showErrorMessage(`Connection lost to ${host}.`);
+                statusBarResource?.dispose();
+                statusBarResource = vscode.window.setStatusBarMessage(`Connection lost to ${host}`);
```
Comment (simsaens) [inline src/aircode.ts:373]: Do we need this twice? I wasn't sure and simply replaced all instances of the previous messages. Seems like this one may be unnecessary? Was going to ask you

File: src/extension.ts [hunk 2]
```diff
@@ -58,6 +81,11 @@ export function activate(context: vscode.ExtensionContext) {
 		else if (host === "") {
 			host = defaultHost;
 		}
+		else {
+			if (!host.includes(":")) {
```
Comment (simsaens) [inline src/extension.ts:85]: Is this from Bri_G's comment? 😀

**Review by simsaens — APPROVED** [review]

File: src/aircode.ts (hunk 1 above)
Comment (jfperusse-bhvr) [reply src/aircode.ts:373]: Oh, actually we can probably get rid of the check there and the duplicate dispose/message.

File: src/extension.ts (hunk 2 above)
Comment (jfperusse-bhvr) [reply src/extension.ts:85]: Well I'm hoping this can help :)

**Pull request merged**